egs/li10/asr1/cmd.sh

```diff
@@ -0,0 +1,16 @@
+# you can change cmd.sh depending on what type of queue you are using.
+# If you have no queueing system and want to run on a local machine, you
+# can change all instances 'queue.pl' to run.pl (but be careful and run
+# commands one by one: most recipes will exhaust the memory on your
+# machine).  queue.pl works with GridEngine (qsub).  slurm.pl works
+# with slurm.  Different queues are configured differently, with different
+# queue names and different ways of specifying things like memory;
+# to account for these differences you can create and edit the file
+# conf/queue.conf to match your queue's configuration.  Search for
+# conf/queue.conf in http://kaldi-asr.org/doc/queue.html for more information,
+# or search for the string 'default_config' in utils/queue.pl or utils/slurm.pl.
+
+export train_cmd="queue.pl --mem 2G"
+export cuda_cmd="queue.pl --mem 8G --gpu 1 --config conf/gpu.conf"
```

Comment:
this is the JHU conf isn't it?  it is nice to be the same to the other egs https://github.com/sw005320/espnet-1/blob/f553100c2c96c8715062f5bc5b168dffb3dc5a4f/egs/wsj/asr1/cmd.sh

Reply:
done